tests/test_utils.py

```diff
@@ -0,0 +1,86 @@
+import ast
+
+from . import skip_if_not_has_async
+from vulture import utils
+
+
+def check(code, expected_names):
```

Comment:
I think this function should be renamed to `check_decorator_names`, do you agree @jendrikseipp?

Reply:
Yes, please :-)